# from django.core.exceptions import ValidationError


class ListingValidator:
    def __init__(self): ...

    def __call__(self):
        return True


class ListingimageValidator:
    def __init__(self): ...

    def __call__(self):
        return True
